src/main/java/org/nrnb/gsoc/enrichment/ui/EnrichmentTableModel.java

```diff
@@ -260,6 +260,96 @@ private boolean inSource(List<TermSource> sources, String termName) {
         return false;
     }
 
+    public void removeRedundantTerms(List<TermSource> sources, double cutoff) {
+        List<CyRow> rows = cyTable.getAllRows();
+        Long[] rowArray = new Long[rows.size()];
+        int i = 0;
+        for (CyRow row : rows) {
```

Comment:
Optional Change: Why not write the loop as a simpler for loop. 
```
for(int i=0;i<rows.size();i++)
```
The code may be simpler.

Reply:
The `i` is actually not just for iteration, it is used to keep row count. Renamed variable it to `rowCount` now.